spec/system/task_spec.rb

```diff
@@ -85,29 +89,31 @@
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
         # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
-        visit edit_project_task_path(project, task)
+        task_done = FactoryBot.create(:task, :done,  project_id: project.id)
+        visit edit_project_task_path(project, task_done)
         select 'todo', from: 'Status'
         click_button 'Update Task'
         expect(page).to have_content('todo')
         expect(page).not_to have_content(Time.current.strftime('%Y-%m-%d'))
-        expect(current_path).to eq project_task_path(project, task)
+        expect(current_path).to eq project_task_path(project, task_done)
       end
     end
   end
 
   describe 'Task削除' do
+    let(:project){ create(:project) }
+    let!(:task){ create(:task, project_id: project.id) }
+
     context '正常系' do
       # FIXME: テストが失敗するので修正してください
-      xit 'Taskが削除されること' do
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
+      it 'Taskが削除されること' do
         visit project_tasks_path(project)
         click_link 'Destroy'
         page.driver.browser.switch_to.alert.accept
-        expect(page).not_to have_content task.title
-        expect(Task.count).to eq 0
+        within('tbody') do
+          expect(page).not_to have_content task.title
+          expect(Task.count).to eq 0
+        end
```

Comment:
画面上の要素をfindする形式でも可能です！
`describe 'Task編集'`でもこの書き方で要素を指定していますね。
```suggestion
        expect(find('.task_list').not_to have_content task.title
        expect(Task.count).to eq 0
```

Reply:
a2d5347 にて修正しました